.env.sample

```diff
@@ -1,3 +1,8 @@
 DATABASE_URL=postgres://postgres:password@localhost:5432/teamwork
 PORT=3000
-TOKEN_SECRET=secret
\ No newline at end of file
+TOKEN_SECRET=secret
+EMAIL_HOST=smtp.gmail.com
+EMAIL_PASSWORD="your host password"
+EMAIL_PORT=465
+EMAIL_USERNAME=example@gmail.com
+EMAIL_FROM=businessexample@gmail.com
```

Comment:
kindly add the production envs here as well so we don't have to go through the code to see the fields to use

Reply:
done @jocrah